src/wallet/wallet2.cpp

```diff
@@ -8670,36 +8683,21 @@ uint64_t wallet2::import_key_images(const std::vector<std::pair<crypto::key_imag
 			spent += amount;
 		else
 			unspent += amount;
+
 		LOG_PRINT_L2("Transfer " << i << ": " << print_money(amount) << " (" << td.m_global_output_index << "): "
```

Comment:
Before the loop it should be a exception if `req.key_images.size() > signed_key_images,size()`